src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
+import WeatherDetails from './Components/WeatherDetails/WeatherDetails';
+
+class App extends Component {
+  state = {
+    weatherData: '',
+  };
+
+  handleState = data => {
+    this.setState({
+      weatherData: data,
+    });
+  };
+
+  render() {
+    const { weatherData } = this.state;
+
+    return (
+      <>
+        <Header />
+        <div className="container">
+          <Form handleState={this.handleState} />
+          {weatherData && <WeatherDetails apiRequestData={weatherData} />}
```

Comment:
name prop just 
`data={weatherData}`

giving info that the data is coming from an external API is not necessary